cl/lib/elasticsearch_utils.py

```diff
@@ -515,15 +521,28 @@ def build_sort_results(
         "citeCount asc": {"citeCount": {"order": "asc"}},
     }
 
-    if api_version == "v4":
-        # Override dateFiled sorting keys in V4 API to work alongside the custom
-        # function score for sorting by dateFiled.
+    if api_version == "v4" and cd["type"] in [
+        SEARCH_TYPES.RECAP,
+        SEARCH_TYPES.DOCKETS,
+        SEARCH_TYPES.RECAP_DOCUMENT,
+    ]:
+        # Override dateFiled sorting keys in V4 RECAP Search API to work
+        # alongside the custom function score for sorting by dateFiled.
         order_by_map["dateFiled desc"] = {"_score": {"order": "desc"}}
         order_by_map["dateFiled asc"] = {"_score": {"order": "desc"}}
 
-    if toggle_sorting and api_version == "v4":
-        # Override the sorting keys in the V4 API when toggle_sorting is True
-        # for backward cursor pagination based on fields that use a custom
+    if (
+        toggle_sorting
+        and api_version == "v4"
+        and cd["type"]
+        in [
+            SEARCH_TYPES.RECAP,
+            SEARCH_TYPES.DOCKETS,
+            SEARCH_TYPES.RECAP_DOCUMENT,
+        ]
```

Comment:
Let's refactor this code to store the `membership` check in a boolean variable. We can call it `is_recap_search` and reuse it in both if statements.

```python 
def build_sort_results:
    ...
    
    is_recap_search = cd["type"] in [
        SEARCH_TYPES.RECAP,
        SEARCH_TYPES.DOCKETS,
        SEARCH_TYPES.RECAP_DOCUMENT,
    ]

    if api_version == "v4" and is_recap_search:
        ...

    if (
        toggle_sorting
        and api_version == "v4"
        and is_recap_search
    ):
        ...

```

Reply:
Great! I've applied the suggestion and named the variable: `require_v4_function_score` since it also includes PEOPLE in #4021